package proto_ff;

import "yd_fieldoptions.proto";


message tasktask
{
	optional int32 id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 minlv = 2[(yd_fieldoptions.field_cname) = "发放等级"];
	optional int32 truelv = 3[(yd_fieldoptions.field_cname) = "可完成等级"];
	optional int32 type = 4[(yd_fieldoptions.field_cname) = "任务类型"];
	optional string showtype = 5[(yd_fieldoptions.field_cname) = "显示类型", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 weights = 6[(yd_fieldoptions.field_cname) = "排序权重"];
	optional string name = 7[(yd_fieldoptions.field_cname) = "任务名", (yd_fieldoptions.field_bufsize) = 60];
	optional string rececond = 8[(yd_fieldoptions.field_cname) = "接取条件", (yd_fieldoptions.field_bufsize) = 60];
	optional string receadd = 9[(yd_fieldoptions.field_cname) = "过程", (yd_fieldoptions.field_bufsize) = 60];
	optional string comcond = 10[(yd_fieldoptions.field_cname) = "完成条件", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 nexttask = 11[(yd_fieldoptions.field_cname) = "后续任务"];
	optional string tracker = 12[(yd_fieldoptions.field_cname) = "任务追踪", (yd_fieldoptions.field_bufsize) = 60];
	optional string subtype = 13[(yd_fieldoptions.field_cname) = "提交方式", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 subnpc = 14[(yd_fieldoptions.field_cname) = "npc的ID"];
	optional string subdesc = 15[(yd_fieldoptions.field_cname) = "提交描述", (yd_fieldoptions.field_bufsize) = 200];
	optional int32 subaward = 16[(yd_fieldoptions.field_cname) = "提交奖励"];
	repeated string submsg = 17[(yd_fieldoptions.field_cname) = "提交对话", (yd_fieldoptions.field_arysize) = 4, (yd_fieldoptions.field_bufsize) = 300];
}


message Sheet_tasktask
{
	repeated tasktask tasktask_List = 1[(yd_fieldoptions.field_arysize)=200];
}
